### DISCUSSION TITLE
Does this platform support RethinkDB?

### DISCUSSION THREAD

**Discussion by TheTechRobo** [Budibase/budibase#5957] [Help]
I mainly use RethinkDB because I'm used to it. Does this platform support it? (I don't mean the changefeeds: while that would be a useful feature, I'm just wondering if there is any barebones support.)

**Comment by melohagan** [comment]
Hey @TheTechRobo 

There is no RethinkDB integration, but feel free to request it. 

As a realtime database, the Redis or Firebase integrations may be suitable. 

If it really is purely out of comfort, the BudibaseDB is a straightforward NoSQL DB, easy to use, and requires no setup at all.
